library/std/src/sync/mutex.rs

```diff
@@ -183,8 +183,33 @@ pub struct Mutex<T: ?Sized> {
 
 // these are the only places where `T: Send` matters; all other
 // functionality works fine on a single thread.
+
+/// SAFETY
+///
+/// The `Send` and `Sync` implementations for `Mutex` ensure that it is safe to
+/// share instances of `Mutex` between threads when the protected data is also
+/// thread-safe. The following explains the safety guarantees:
```

Comment:
This part explains what `Sync` & `Send` do, which is already explained in `Sync` & `Send` doc.

I think SAFETY comments should be as short as possible, so this part should be removed.

Reply:
okay done